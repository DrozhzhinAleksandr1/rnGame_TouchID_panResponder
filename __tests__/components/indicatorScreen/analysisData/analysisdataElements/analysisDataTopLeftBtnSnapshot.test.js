import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { AnalysisDataTopLeftBtn } from '../../../../../app/components/indicatorScreen/analysisData/analysisDataElements/analysisDataTopLeftBtn';

it('create snapshot AnalysisDataTopLeftBtn', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<AnalysisDataTopLeftBtn quotationText={'Котировка:'} quotationNum={'0.94527'}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});
