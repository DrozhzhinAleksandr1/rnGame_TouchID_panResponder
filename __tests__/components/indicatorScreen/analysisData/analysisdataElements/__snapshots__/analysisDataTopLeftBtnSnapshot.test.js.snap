// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot AnalysisDataTopLeftBtn 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontFamily": "Roboto-Light",
        "fontSize": 11,
        "lineHeight": 13,
        "marginBottom": -5,
        "opacity": 0.5,
      }
    }
  >
    Котировка:
  </Text>
  <Component
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#fff",
            "fontFamily": "Roboto-Light",
            "fontSize": 20,
          },
          Object {
            "paddingTop": 3,
          },
        ]
      }
    >
      0
      .
      9
      4
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#fff",
            "fontFamily": "Roboto-Light",
            "fontSize": 20,
          },
          Object {
            "fontSize": 23,
          },
        ]
      }
    >
      5
      2
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#fff",
            "fontFamily": "Roboto-Light",
            "fontSize": 20,
          },
          Object {
            "paddingTop": 3,
          },
        ]
      }
    >
      7
    </Text>
  </Component>
</Component>
`;
